Apps/Sandcastle/gallery/Terrain.html

```diff
@@ -136,6 +137,7 @@
 var terrainSamplePositions;
 
 function sampleTerrainSuccess() {
+    viewer.scene.globe.depthTestAgainstTerrain = true;
```

Comment:
The depth test is only necessary in this example when we add the billboards to the scene.  Putting it up top disconnected it from its use and also made it look like it was a required part of Terrain setup.